src/components/VLink.vue

```diff
@@ -16,14 +16,18 @@
  */
 import { computed, defineComponent, useContext } from '@nuxtjs/composition-api'
 
-const defaultProps = { target: '_blank', rel: 'noopener noreferrer' }
+const defaultProps = Object.freeze({
+  target: '_blank',
+  rel: 'noopener noreferrer',
+})
 
 export default defineComponent({
   name: 'VLink',
   props: {
     href: {
       type: String,
-      validator: (v) =>
+      required: false,
+      validator: (v: unknown) =>
```

Comment:
Please note that setting `validator`'s argument explicitly to `unknown` is necessary for prop extraction to work when a `validator` is present. Even though `unknown` is the inferred type. :shrug: